docs/src/main/paradox/scala/http/common/marshalling.md

```diff
@@ -1,9 +1,9 @@
 # Marshalling
 
-"Marshalling" is the process of converting a higher-level (object) structure into some kind of lower-level
-representation, often a "wire format". Other popular names for it are "Serialization" or "Pickling".
+Marshalling is the process of converting a higher-level (object) structure into some kind of lower-level
+representation, often a "wire format". Other popular names for marshalling are "serialization" or "pickling".
```

Comment:
I'm not so sure about not capitalizing the quoted serialization and pickling. But maybe is the German influence that biases me.